ports/matio/vcpkg.json

```
@@ -1,6 +1,6 @@
 {
   "name": "matio",
-  "version": "1.5.24",
+  "version": "1.5.25",
```

```suggestion
  "version": "1.5.26",
```
https://github.com/tbeu/matio/releases/tag/v1.5.26